lib/messages.d.ts

```diff
@@ -0,0 +1,13 @@
+//
+// automatically generated code; do not edit manually!
+//
+///<reference path=".d.ts"/>
+interface IMessages{
```

Comment:
We've generated the same exact interface in common-lib - https://github.com/telerik/mobile-cli-lib/pull/484/files#diff-2f056d897a840743988e1666c5f9b1baR5. Can we reuse it?